### ISSUE TITLE
Import from scapy.sendrecv in Windows fails

### ISSUE THREAD

**Issue by martingalloar** [secdev/scapy#111]
Importing from `scapy.sendrecv` fails in Windows:

```
PS Microsoft.PowerShell.Core\FileSystem::\\vboxsrv\workspace\pysap> ipython
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)]
Type "copyright", "credits" or "license" for more information.

IPython 4.1.1 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: from scapy.sendrecv import sniff
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-56f9d4ea2ee6> in <module>()
----> 1 from scapy.sendrecv import sniff

C:\tools\python2\lib\site-packages\scapy\sendrecv.py in <module>()
     11 from select import select
     12 from data import *
---> 13 import arch
     14 from config import conf
     15 from packet import Gen

C:\tools\python2\lib\site-packages\scapy\arch\__init__.pyc in <module>()
     78     from solaris import *
     79 elif WINDOWS:
---> 80     from windows import *
     81
     82 if scapy.config.conf.iface is None:

C:\tools\python2\lib\site-packages\scapy\arch\windows\__init__.py in <module>()
     15 from scapy.base_classes import Gen, Net, SetGen
     16 import scapy.plist as plist
---> 17 from scapy.sendrecv import debug, srp1
     18 from scapy.layers.l2 import Ether, ARP
     19 from scapy.data import MTU, ETHER_BROADCAST, ETH_P_ARP

ImportError: cannot import name debug
```

I think there's some kind of import cycle, as `sendrecv` is architecture-dependent but the Windows arch requires to import `sendrecv` module at the same time. In the past we've fixed a [similar issue](https://bitbucket.org/secdev/scapy/pull-requests/56/fixed-import-of-supersocket-module-on-non/diff) in `supersocket`, but I'm not sure what's the best approach to fix this case.

Any idea on this?

**Comment by martingalloar** [comment]
I can workaround this in my code by doing `import arch` before the `from scapy.sendrecv import sniff`, but library-wise it might be good to find another solution.

**Comment by p-l-** [comment]
Thanks for this report! That's totally a bug and needs to be fixed.

**Comment by ronanpaixao** [comment]
There appears to be many dependency loops on Windows. There are many cases, for example:

    from scapy.sendrecv import debug
    
```
<python dir>\lib\site-packages\scapy\arch\windows\__init__.py in <module>()
     15 from scapy.base_classes import Gen, Net, SetGen
     16 import scapy.plist as plist
---> 17 from scapy.sendrecv import debug, srp1
     18 from scapy.layers.l2 import Ether, ARP
     19 from scapy.data import MTU, ETHER_BROADCAST, ETH_P_ARP

ImportError: cannot import name debug
```

**Comment by ronanpaixao** [comment]
By the way, this is version 2.3.2.

**Comment by robin-jarry** [comment]
I think that PR #293 fixes this problem.

**Comment by p-l-** [comment]
Fixed by @robin-jarry in #293

**Comment by martingalloar** [comment]
I'm still having troubles with scapy 2.3.3 in Windows:

```
PS C:\devel> python -V
Python 2.7.11
PS C:\devel> python -c "import scapy;print(scapy.VERSION)"
2.3.3
PS C:\devel> python -c "from scapy.sendrecv import debug"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\tools\python2\lib\site-packages\scapy\sendrecv.py", line 14, in <module>
    from scapy.arch.consts import DARWIN, FREEBSD
  File "C:\tools\python2\lib\site-packages\scapy\arch\__init__.py", line 84, in <module>
    from scapy.arch.windows.compatibility import *
  File "C:\tools\python2\lib\site-packages\scapy\arch\windows\compatibility.py", line 156, in <module>
    scapy.sendrecv.sndrcv = sndrcv
AttributeError: 'module' object has no attribute 'sendrecv'
```

While the following works fine:

`PS C:\devel> python -c "import scapy.arch;from scapy.sendrecv import debug"`

Maybe there are more circular imports out there?
